docs/general-concepts/icons.md

```diff
@@ -20,10 +20,13 @@ To display such an icon you can use:
 <span class="icon-cancel"></span> <?php echo Text::_('JCANCEL') ?> 
 ```
 
-You can also display FontAwesome icons which aren't used in Joomla, but you have to include the "fa" class:
+You can also display FontAwesome icons which aren't used in Joomla, but you have to include an additional CSS class.
+In the past this has been "fa" and then after that "fas" or "fab", 
+but according to [fa style names](https://docs.fontawesome.com/web/setup/upgrade/whats-changed#full-style-names)
+the CSS classes should be "fa-solid" or "fa-brands":
```

Comment:
```suggestion
You can also display FontAwesome icons which aren't used in Joomla, but you have to include an additional CSS class.
According to [fa style names](https://docs.fontawesome.com/web/setup/upgrade/whats-changed#full-style-names) the CSS classes should be "fa-solid" or "fa-brands":
```

If you like to have this "in the past" then please add it to the migration section.

Or as we discussed at some point add a todo marker/list so we can remove this on the next major release.